### ISSUE TITLE
Run analysis in parallel

### ISSUE THREAD

**Issue by wandadk** [RETURN-project/makeDataCube#18]
## Problem

Currently, the code is not fully run in parallel. It should be optimized in various ways:

## In `make_Landsat_cube.Rmd`

1. Several parts could be run in parallel (eg downloading Landsat data) and are currently sequential
2. Processing data in FORCE can be run in parallel without problems (the settings file should be adjusted), but we should define which settings are most optimal to process the data. More info can be found [here](https://force-eo.readthedocs.io/en/latest/components/lower-level/level2/level2.html#level2-bulk) and in section 8 of this [tutorial](https://davidfrantz.github.io/tutorials/force-ard/l2-ard/)

## In `make_mask_fire_cube.Rmd`

1. A loop is used to iterate over each tile. In each iteration, data are prepared. It should be fairly easy to parallelize this code. One point of attention is reading and writing of data (e.g. temporary files are generated).

**Comment by PabRod** [comment]
## Proposed solutions

## In `make_Landsat_cube.Rmd`

1. Use Snakemake for streamlining the workflow.
2. Use FORCE's internal parallelization. Practically, this just translates to passing `NPROC` and `NTHREAD` to the parameter file. Subtask: figure out the advisable values.

## In `make_mask_fire_cube.Rmd`

1. Explore the possibility of [parallelizing using R](https://blog.esciencecenter.nl/parallel-r-in-a-nutshell-4391d45b5461), as we did with [`BenchmarkRecovery`](https://github.com/RETURN-project/BenchmarkRecovery).

**Comment by PabRod** [comment]
Merge into #36
